ortools/linear_solver/linear_solver.cc

```diff
@@ -1435,7 +1452,9 @@ void MPSolver::GenerateConstraintNameIndex() const {
   }
 }
 
+#if defined(USE_GUROBI)
```

Comment:
Revert this protection. This is a virtual method that is only implemented for Gurobi. No need for a macro protection too.